### DISCUSSION TITLE
[Info needed] Infinite TTL for webapp

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#4493] [Q&A] [answered]
I actually have a question,
Some self hosted services provide a way to disable auto disconnection after some time.

Us who actually use an SSO on all our domain are "already" covered security whise, so i would like to know how i can indefinitly maintain one connection until a way to disable to form come like asked in this issue: https://github.com/AdguardTeam/AdGuardHome/issues/4426

What the maximum number of time accepted by adguard in the yaml file : web_session_ttl
Does 0 disable the auto disconnect ?

Thanks

**Comment by ainar-g** [comment, marked as answer]
If you mean that you would like to have an infinite web session, you can actually just set `web_session_ttl` to a very large value.  Setting it to something like `1000000` would give you a session more than a century long, which should be plenty for most tasks.
